packages/cashier/src/stores/payment-agent-transfer-store.js

```diff
@@ -45,7 +45,13 @@ export default class PaymentAgentTransferStore {
     }
 
     async checkIsPaymentAgent() {
-        const get_settings = (await this.WS.authorized.storage.getSettings()).get_settings;
+        const { client, ui } = this.root_store;
+        const { account_settings } = client;
+        const { is_real_acc_signup_on } = ui;
```

Comment:
```suggestion
        const { client: { account_settings }, ui: { is_real_acc_signup_on } } = this.root_store;
```
@jim-deriv nit-pick😉

Reply:
@maryia-deriv , looks good :+1:
 but isn’t Jim's implementation more readable?
 we use this approach everywhere btw

Reply:
@maryia-deriv, since it went through `QA` and it's ready, I request that I don't touch it and just let it get merged. Thank you